Task2 code

breligor opened the pull request:
Сделать крестики в таком виде, как мы обсуждали в субботу, пока не получилось. Много непонятных моментов, на которые самостоятельно не смог найти ответы. Нужно еще время - буду доделывать. Пока присылаю вариант 3x3 с проверкой на ничью, защитой от повторного ввода и небольшой адаптацией под 450 px.

breligor committed:
Task2

start

breligor committed:
start

breligor committed:
game field on html

breligor committed:
TTT 3x3

Крестики нолики 3x3
Версия немасштабируемая

breligor committed:
Delete main.css

breligor committed:
адаптация под 450px

breligor committed:
Merge branch 'Task2_code' of https://github.com/breligor/Task2_Brel_Igor into Task2_code

breligor committed:
исправлена ошибка определения ничьей

breligor committed:
шрифт X/0 под 450 px уменьшен в @media

breligor committed:
исправлена ошибка

после объявления выигрыша можно было дальше вводить в поля значения x или 0 и поменять победителя на противоположного

6ygga commented on script.js:
```diff
@@ -0,0 +1,73 @@
+                doResult(result);
+                return
+            }
+
```
убери лишнюю строку

6ygga commented on script.js:
```diff
@@ -0,0 +1,73 @@
+            step++;
+            checkWinner();
+        }
+
```
убери лишнюю строку

6ygga commented on index.html:
```diff
@@ -0,0 +1,21 @@
+    <div id="resultMessage"></div>
+    <button id="myBtn" onclick="location.reload()">Новая игра</button>
+</div>
+
```
лишняя строка

6ygga commented on style.css:
```diff
@@ -0,0 +1,87 @@
+    display: flex;
+    justify-content: center;
+    text-align: center;
+
```
лишняя строка

6ygga commented on style.css:
```diff
@@ -0,0 +1,87 @@
+    text-align: center;
+
+}
+.container {
```
а вот после каждого класса надо делать пропуск

breligor committed:
bug fixed

исправлена ошибка в результате который после окончания игры и объявления победителя оставалась возможность дальше вставлять X/0 в поле и изменить победителя на противоположного
